versioned_docs/version-latest/protocol/substrate-cli.mdx

```diff
@@ -606,209 +583,221 @@ Download the Executable Files, using the appropriate commands:
     </summary>
 		Node:
 		<CodeBlock language="shell-session">
-		wget -O /root/.local/bin/subspace-node https://github.com/subspace/subspace/releases/download/gemini-3g-2023-sep-25/subspace-node-ubuntu-aarch64-gemini-3g-2023-sep-25
+		wget -O subspace-node https://github.com/subspace/subspace/releases/download/gemini-3g-2023-nov-07/subspace-node-ubuntu-aarch64-gemini-3g-2023-nov-07
 		</CodeBlock>
 		Farmer:
 		<CodeBlock language="shell-session">
-		wget -O /root/.local/bin/subspace-farmer https://github.com/subspace/subspace/releases/download/gemini-3g-2023-sep-25/subspace-farmer-ubuntu-aarch64-gemini-3g-2023-sep-25
+		wget -O subspace-farmer https://github.com/subspace/subspace/releases/download/gemini-3g-2023-nov-07/subspace-farmer-ubuntu-aarch64-gemini-3g-2023-nov-07
 		</CodeBlock>
 </details>
 
-By default, these commands download the executable files to the `/root/.local/bin` directory. 
-You can change the directory by modifying the respective string.
-
-You can use the same commands to update the node.
-
-Make the Files Executable:
-```shell-session
-chmod +x /root/.local/bin/subspace-node
-chmod +x /root/.local/bin/subspace-farmer
+Copy them to `/usr/local/bin` and delete originals:  
+```console
+sudo cp subspace-node subspace-farmer /usr/local/bin
 ```
+Make them executable:
+```console
+sudo chmod +x /usr/local/bin/subspace-node /usr/local/bin/subspace-farmer
+```
+*However, you can download binaries directly to the right place if you run these commands from `/usr/local/bin` directory with root privilegies.*
 
-Create a Directory for Node and Farmer Data:
-```shell-session
-mkdir -p /root/.local/share
+Set ownership to user 'subspace':
+```console
+sudo chown subspace:subspace /usr/local/bin/subspace-node
+sudo chown subspace:subspace /usr/local/bin/subspace-farmer
 ```
 
-</TabItem>
+:::tip You can use the same commands for updating the node and the farmer
+:::
 
-<TabItem value="regular_user" label="👤 Regular User">
-Create a New User for Subspace:
 
-```shell-session
-sudo useradd -m -p ! -s /sbin/nologin -c "" subspace
+### Creating dedicated directories
+This command creates `subspace-node` and `subspace-farmer` directories for node's data and farmer's data respectively with owning by user 'subspace':
+```console
+sudo -u subspace mkdir /var/lib/subspace-node /var/lib/subspace-farmer
```

Comment:
I understand what you mean. But I just didn't include it in my PR, but on my local repo I created a directory /var/lib/subspace, and in it there are two more subdirectories, subspace-node and subspace-farmer, and this upper directory (subspace) has an owner - user subspace, and this scheme is quite handy.

Reply:
I agree with @extSunset comment, I don't think we should use examples of adding new folders in the root location, since some users won't have permissions to do so and won't be able to complete the further instructions.

Reply:
> I agree with @extSunset comment, I don't think we should use examples of adding new folders in the root location, since some users won't have permission to do so and won't be able to complete the further instructions.

What do you mean? Did you mean new folders in `/var/lib` ?

Reply:
> > I agree with @extSunset comment, I don't think we should use examples of adding new folders in the root location, since some users won't have permission to do so and won't be able to complete the further instructions.
> 
> What do you mean? Did you mean new folders in `/var/lib` ?

Exactly.

Reply:
I'm not arguing. i see the benefits of it. Thank you guys 🙏